process/block/interceptedBlocks/interceptedMetaBlockHeader.go

```diff
@@ -91,6 +94,15 @@ func (imh *InterceptedMetaHeader) CheckValidity() error {
 		if err != nil {
 			return err
 		}
+
+		if imh.isMetaHeaderEpochOutOfRange() {
+			// TODO: remove this log after testing
```

Comment:
I think this log can remain as it would be helpful in the future for debugging purposes

Reply:
We can keep it as a log.Trace because we might end up printing too much when a node will voluntary send bad data